Added necessary config to get django to work with apache

lucas-ps opened the pull request.

lucas-ps committed:
added necessary config to get django to work with apache

Conor-Behard333 committed:
Merge branch 'main' into server-config

Conor-Behard333 reviewed (changes requested):
Please add a description of what you have changed and what needs to be tested (if needs to be)
and make a card on trello for this PR

lucas-ps commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Implement program on google cloud](https://trello.com/c/3jeLEfnX/132-implement-program-on-google-cloud)

lucas-ps commented:
I've added a few modules and changed some config to allow for the server to work with apache, including how static and media files work, and implementing mod_wsgi. Shouldn't need any testing as I haven't really changed anything other than some configurations.
